db/re/item_db.conf

```diff
@@ -61939,6 +61940,11 @@ item_db: (
 	Slots: 1
 	Loc: 256
 	View: 525
+	Script: <"
+		bonus bDex, 2;
+		skill TF_DOUBLE, 5;
```

Comment:
That is the way it works once, some years ago, but somebody reported it as a bug, an also add the skill enabling into skill tree and not only give bDoubleRate bonus.

You can check this commits about it: e1cd46aacaab8a0e398d809cfda67aebc3cc77f2, 137096757561eb466f2bc7cbb89d65ca70ba22a3

Issue 5935 it's pretty informative: http://herc.ws/board/tracker/issue-5935-sidewinder-card-effect The other one I couldn't find it. Anyway, maybe those people are wrong too.